### ISSUE TITLE
About HUD.

### ISSUE THREAD

**Issue by HakkaBats** [sd805/l4d2vr#66]
I am using v0.6. At present, I can see the HUD, but the size is not right. It is far away and always on the upper side. How can I modify it to fit my screen? What does this relate to?

**Comment by Dirtycleaner** [comment]
Hi, have a same issue. Till game didnt get Last update everything worked. Now in ver 0.5 and 0.6 i cant see ingame hud too..

**Comment by sd805** [comment]
@HakkaBats You can change the HUD distance in VR/config.txt, but the default should be pretty close. Not sure what's causing the height issue. Does the main menu look high up too?

@Dirtycleaner Are you able to see the main menu?

**Comment by Dirtycleaner** [comment]
> 

Yes

**Comment by Dirtycleaner** [comment]
> @HakkaBats You can change the HUD distance in VR/config.txt, but the default should be pretty close. Not sure what's causing the height issue. Does the main menu look high up too?
> 
> @Dirtycleaner Are you able to see the main menu?



> @HakkaBats You can change the HUD distance in VR/config.txt, but the default should be pretty close. Not sure what's causing the height issue. Does the main menu look high up too?
> 
> @Dirtycleaner Are you able to see the main menu?

Yes

**Comment by HakkaBats** [comment]
@sd805 Yes, the main menu looks high up, I think VR/config.txt should also support the adjustment of HUD height.

**Comment by Dirtycleaner** [comment]
> @HakkaBats  Have you tried to adjust hud distance in config?

**Comment by HakkaBats** [comment]
@sd805 Yes, I tried, but it was always at the original height.

**Comment by sd805** [comment]
@HakkaBats I noticed that if you override world scale in SteamVR settings, it messes up the height of the HUD. Maybe that's the issue?

**Comment by HakkaBats** [comment]
@sd805 But my override world scale status is closed, I tried to set it to the maximum value. The HUD height was normal, but the vision became quite strange.

**Comment by optimumbox** [comment]
I also want to add that the hud is not functioning correctly for me. I appears giant and in the way of my fov at all times no matter what the settings are in the ini. I even tried using a hud removal mod and it still just resulted in a giant box blocking most of my view. I've attached what I see in game. It's the same issue even with the hud turned on. The main difference is the health bars aren't visible.
![20221220014813_1](https://user-images.githubusercontent.com/24198612/208724243-c22a1917-335f-4a7d-86e6-170907b11e27.jpg)

**Comment by Hoshi82** [comment]
Same issue here on my Valve Index on SteamVR. The menu and hud is always way up and quite a bit on the right side, not centered.

**Comment by HakkaBats** [comment]
V0.6.1 has fixed the problem I encountered. Although it is still a little short, thank very much.
